src/index.js

```diff
@@ -64,6 +66,27 @@ async function bundleMDX(
           if (fullModulePath in absoluteFiles) return {path: fullModulePath}
         }
 
+        if (cwd) {
+          const cwdModulePath = modulePath.replace(
+            /^(.*__mdx_bundler_fake_dir__)/,
+            cwd,
+          )
+
+          for (const ext of [
+            '',
+            '.js',
+            '.ts',
+            '.jsx',
+            '.tsx',
+            '.json',
+            '.mdx',
+          ]) {
+            const fullCWDModulePath = `${cwdModulePath}${ext}`
+            if (fs.existsSync(fullCWDModulePath))
+              return {path: fullCWDModulePath}
```

Comment:
If it's multiple lines, I prefer the `{`:

```suggestion
            if (fs.existsSync(fullCWDModulePath)) {
              return {path: fullCWDModulePath}
            }
```

But actually, what would happen if we didn't have this check at all? Wouldn't esbuild fail with a reasonable error message anyway?

```suggestion
            return {path: fullCWDModulePath}
```

Reply:
This is all gone now :)

I forgot that if you return null from an `onResolve` callback in esbuild it will try another plugin. I've added a `inMemory` flag to the `pluginData` so that we can pick up on it in the `onLoad` step as I now don't use the fake directory if `cwd` is specified.